app/serializers/game_serializer.rb

```diff
@@ -11,6 +11,6 @@ def player_2_board
   end
 
   def message
-    @instance_options[:message]
+    object.messages
```

Comment:
we might need to move this back to the @instance_options since these messages live on the TurnProcesser and the messages that are in there might be missed with this configuration. We can use this for now though!

Reply:
Good catch - I will put this back in there!